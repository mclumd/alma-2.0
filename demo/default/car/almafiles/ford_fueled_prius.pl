rel(gas_tank_car, ford).
not(rel(fueled, ford)).

rel(electric_car, prius).
rel(gas_tank_car, prius).
rel(charged, prius).
rel(fueled, prius).
