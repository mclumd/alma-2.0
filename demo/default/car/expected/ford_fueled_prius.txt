rel(car, ford)
rel(gas_tank_car, ford)
~rel(powered, ford)
~rel(fueled, ford)
abnormal(ford, car, "~rel(powered, ford)")
rel(car, prius)
rel(gas_tank_car, prius)
rel(electric_car, prius)
rel(charged, prius)
rel(fueled, prius)
rel(powered, prius)
